QUARKUS-451.md

```diff
@@ -0,0 +1,42 @@
+# QUARKUS-451 - Testing and verifying OpenShift Kafka with Quarkus
+
+The goal of this test plan is to cover the main points of conflicts between Quarkus and Kafka. 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-451
+
+## Scope of the testing
+Following actions were taken to ensure test coverage and automation for reactive kafka applications:
+ - Kafka producer with JSON / AVRO body messages.
+ - Kafka consumer with JSON / AVRO body messages.
+ - Kafka Streams with JSON body messages and windowed.
+ - Was tested over Kafka Strimzi and RH AMQ Streams.
+ - Platform Openshift, Java 11.
+
+### Impact on testsuites and testing automation:
+ - OpenShift TS framework needs to be enhanced to support OpenShift kafka
+   - Would be great if in the future we support some kind of openshift/k8s templating. So install some specifict Kafka (provider/version) would be as simple as apply a template. 
+
+### Impact on resources:
+ - More services will be running on OpenShift cluster, impact on CPU/memory should be minimal (if you don't setup Kafka/zookeper/registry with a persistent volume).
```

Comment:
How many new services will be running with Kafka in the scope?

Kafka is perceived as lightweight but the reality is not so optimistic.

Reply:
Well, that is true, Kafka is not "lightweight":

- 1 Kafka broker (without replication) (755.5 MiB - 343.5 MiB)
- 1 zookeeper () (61.8 MiB)
- 1 Kafka registry (128Mb)